### ISSUE TITLE
Has `offlineimap3` also been abandoned?

### ISSUE THREAD

**Issue by chimbori** [OfflineIMAP/offlineimap3#200]
First there was [`offlineimap`](https://github.com/OfflineIMAP/offlineimap/tags), which was abandoned in favor of [`imapfw`](https://github.com/OfflineIMAP/imapfw/releases).

Then `imapfw` was abandoned, presumably in favor of `offlineimap3`?

Now, `offlineimap3` also appears to have been abandoned?

There has been [no release since October 2021](https://github.com/OfflineIMAP/offlineimap3/releases), not even security-related or compatibility-related releases. And PRs have not been accepted either.

Is the project looking for new maintainers?

Sorry to be asking here, I wasn’t sure where else this discussion could be had. Thank you!

**Comment by chris001** [comment]
You're right, it's wwll time to make a new release.  I'm reaching out to the other maintainers... we'll post back asap.

**Comment by thekix** [comment]
Hello. Yes, you are right, there are no versions, but it is not abandoned. Some pull-requests have been included a few months ago. I'm very busy at the moment, but I'll try to make some time to review it, especially before the new version of Python.

@chris001 we have to see how to manage it.

**Comment by thekix** [comment]
Hi,

I applied some pull requests today. I also created the Telegram group https://t.me/OfflineIMAP to talk about the project and help the users.

Best regards,
kix

**Comment by etbuira** [comment]
If you're unhappy with offlineimap3, you can use a better maintained fork (and drop-in replacement) https://github.com/IMAPMirror/imapmirror

**Comment by chimbori** [comment]
@etbuira Thanks for the fork; looks like you have put in a lot of work in a very short amount of time to bring the project up to date — I appreciate it!

@thekix Any updates about an official OfflineIMAP release? Hope updates are posted here in the open instead of in a closed Telegram group. Those are notoriously hard to search for and follow, and would limit community participation. Development already has been slow, so Telegram is unlikely to bring any velocity improvements. Thanks for all your hard work on this project!

**Comment by thekix** [comment]
Dear @chimbori ,

Last week I checked some issues and applied the patches that were open.
There are currently four pull requests open and I was waiting for feedback on them before applying them. My idea was to apply the patches and release the new version as soon as possible. I will probably do it this weekend.

#154 I am currently checking.
#170 Is include specific exceptions instead bare
#182 and #202 will be included in the same patch, related to setup the project. 

Best regards.
kix

**Comment by chimbori** [comment]
Thank you so much, @thekix ! I lack Python experience, otherwise I would have loved to participate to help maintain this. I appreciate all your hard work over the years!

**Comment by thekix** [comment]
Hi,

I have included all the PR now. The work for the new configuration + testing process took longer than expected.

Please, @chimbori @chris001 could you try the new code?

Regards,
kix

**Comment by keithbowes** [comment]
> If you're unhappy with offlineimap3, you can use a better maintained fork (and drop-in replacement) https://github.com/IMAPMirror/imapmirror

There's also [isync](https://sf.net/p/isync), for those who don't want to bother with Python. offlineimap does have a few advantages (e.g. variables in config files and autorefresh mode) but so does isync (e.g. reasonable TLS connections by default and declarative folder renames).

imapmirror should have been an actual fork of offlineimap (that is, using GitHub's Fork functionality), to make synchronization with offlineimap easier. Keeping it on a separate tree makes it more likely that the two diverge over time (e.g. imapmirror doesn't have my code for finding OpenSSL CA bundles in a distro-agnostic manner, rather than hardcoding it for every distro, many of which aren't hardcoded, making users specify the path manually).

**Comment by etbuira** [comment]
Hi @keithbowes,the whole point of imapmirror is to diverge from offlineimap3 (while keeping on-disk format and config compatibility), so that would be of no real help to have used github's fork. About certificates, the main use i consider is to specify path in config, i might consider using certifi later.

**Comment by nicolas33** [comment]
Hey,

First of all, I'm delighted to see offlineimap3 evolve, thanks to the community and @thekix.

@thekix: I've invited you to the github org maintainers so you're not limited anymore in the github org.


The https://github.com/IMAPMirror/imapmirror project is good news and I hope it's going to have a long life.

I tend to agree with @keithbowes about the technical comment of forking. However, the fork is correct by keeping all the git history. IOW, it's not "github perfect" but the only real issue is about git (it's not about github). The imapmirror fork is perfect in git terms because both share the same history at some point.

**Comment by huyz** [comment]
Hello, I'm trying to decide whether to wait for a new release of offlineimap3 or take the plunge with [IMAPMirror/imapmirror](https://github.com/IMAPMirror/imapmirror).

Any idea when a new release may come out?

**Comment by sudipm-mukherjee** [comment]
@thekix @nicolas33 an update about the future of offlineimap3 will be appreciated. I have mentioned the same at https://github.com/OfflineIMAP/offlineimap3/issues/160#issuecomment-2997529775

**Comment by thekix** [comment]
Hi,

I am closing this issue. Please, continue in #228 

Regards,
kix
